### ISSUE TITLE
APIComparer Console as NuGet package?

### ISSUE THREAD

**Issue by devlead** [ParticularLabs/APIComparer#51]
Just stumbled upon this project on the twitters and it looks really useful :+1: 

Would like to run something like this on build servers as part of their build scripts, is there an existing / planned NuGet package for the Console today?

**Comment by andreasohlund** [comment]
Not at this stage, I actually have some plans to

1. Run it on the buildserver
2. Include the report in the artifacts (we're on TeamCity)
3. Make a ping back to the gihub status api to make sure the link to it included in the "checks" section of each pull request

Let me know if you want to take a stab at it!

**Comment by andreasohlund** [comment]
Side note: We use apiapprover for most of our projects so any changes to the public api is visible in pull requests if you know what to look for

https://github.com/JakeGinnivan/ApiApprover

Example:

https://github.com/Particular/NServiceBus/pull/2890/files#diff-1eb180a4f6d35cd0c53fde55c1a077b3

**Comment by devlead** [comment]
Interesting, I'm one of the maintainers behind @cake-build , see this could be an great potential add-in or tool for us.

I see great benefit for a tool like this, thanks for your time, tips & info :+1:

**Comment by Mpdreamz** [comment]
+1 would love to see the console version on nuget

**Comment by andreasohlund** [comment]
Wouldn't chocolatey be more appropriate since its an "app" ?

**Comment by Mpdreamz** [comment]
Not a big purist but most build related tools end up on nuget (xunit console, paket, gitlink) where as chocolatey is more for desktop app / services IMO :smile:

**Comment by andreasohlund** [comment]
Fair enough, not having to bother with the approval mess that comes with chocolatey would be a win

**Comment by devlead** [comment]
@andreasohlund @Mpdreamz +1 for nuget, as it's almost always available in the build tool chain (i.e. Cake build scripts has "native" support for fetching tools via Nuget). Nothing stopping you from doing both though. Chocolatey is great for the dev machine or when you want it to be available globally.
